apps/web/app/cache/apiKey.ts

```diff
@@ -0,0 +1,22 @@
+"use server";
```

Comment:
Refactor: I created this `cache` folder where all cached queries and revalidation calls can go into